### ISSUE TITLE
[Bug] Direct Keyboard Access not working. Cannot use Keyboard to input text in-game.

### ISSUE THREAD

**Issue by Coffeline** [Ryubing/Issues#78]
### Description of the issue

Direct Keyboard Access does not work.

### Reproduction steps

1. Enable "Direct Keyboard Access" on Settings > Input
2. Run a game that supports native keyboard input
3. Try to type something in-game

### Log file

[Ryujinx_1.3.2_2025-09-05_18-31-52.log](https://github.com/user-attachments/files/22181599/Ryujinx_1.3.2_2025-09-05_18-31-52.log)

### OS

Windows 11

### Ryujinx version

1.3.2

### Game version

2.0.6

### CPU

Ryzen 5 1600

### GPU

Radeon RX570 4GB

### RAM

16GB

### List of applied mods

[Jubbsticks' Updates ](https://gamebanana.com/mods/579817)
[New Leaf OST+](https://gamebanana.com/sounds/60070)

### Additional context?

Currently running the game with mods, but I've tested without mods and the issue persists.
